### PR TITLE
add backend lesson

### PR DESCRIPTION
**Pull request by SaraCubero**

### CODE REVIEW

**Review by Adanzar — APPROVED** [review]

File: challenges/sara/curso/week-6/api/rest-demo-2/controllers/RickController.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    const { heroId } = req.params;
+    const hero = Ricks.find(({ id }) => id === +heroId);
+    if (hero) {
+      res.json(hero);
+    } else {
+      res.status(404);
+      res.send('no encontré el rick');
+    }
+  },
+  postHero: (req, res) => {
```
Comment (gilbecao) [inline challenges/sara/curso/week-6/api/rest-demo-2/controllers/RickController.js:24]: Change function name

Comment (gilbecao) [reply challenges/sara/curso/week-6/api/rest-demo-2/controllers/RickController.js:24]: Change every reference to heroes

File: challenges/sara/curso/week-6/api/rest-demo-2/routes/heroRoutes.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    .get(HeroController.getAll)
+    .post(HeroController.postHero);
+
+  heroRoutes
+    .route('/:heroId')
+    .get(HeroController.getById)
+    .put(HeroController.putHero)
+    .delete(HeroController.deleteHero);
+
+  heroRoutes
```
Comment (gilbecao) [inline challenges/sara/curso/week-6/api/rest-demo-2/routes/heroRoutes.js:19]: Move to a new file and call from index

**Review by gilbecao — CHANGES_REQUESTED** [review]
Review comments

**Review by gilbecao — APPROVED** [review]

**Pull request merged**